addon/validations/factory.js

```diff
@@ -430,7 +430,7 @@ function createTopLevelPropsMixin(validatableAttrs) {
 function getCPDependentKeysFor(attribute, validations, owner) {
   let dependentKeys = validations.map(validation => {
     const type = validation._type;
-    const options = validation.options;
+    const options = getWithDefault(validation, 'options', Ember.Object.create());
```

Comment:
`validation.options` is valid here. This should be an will always be a pojo. Please revert this change back.